docs/en/engines/table-engines/integrations/embedded-rocksdb.md

```diff
@@ -39,4 +39,38 @@ ENGINE = EmbeddedRocksDB
 PRIMARY KEY key
 ```
 
+## Metrics
+
+There is also `system.rocksdb` table, that expose rocksdb statistics:
+
+```sql
+SELECT
+    name,
+    value
+FROM system.rocksdb
+
+┌─name──────────────────────┬─value─┐
+│ no.file.opens             │     1 │
+│ number.block.decompressed │     1 │
+└───────────────────────────┴───────┘
+```
+
+## Configuration
+
+You can also change any [rocksdb options](https://github.com/facebook/rocksdb/wiki/Option-String-and-Option-Map) using config:
+
+```xml
+<rocksdb>
+    <max_background_jobs>8</max_background_jobs>
+</rocksdb>
+```
+
+Or percisely for one table:
+
+```xml
+<rocksdb_TABLE_NAME>
```

Comment:
It is strange (string concatenation, unstructured data in config).
Maybe better:

```
<rocksdb>
    <max_background_jobs>8</max_background_jobs>
    <tables>
        <name1>
            <max_background_jobs>1</max_background_jobs>
        </name1>
        ...
    </tables>
</rocksdb>
```

Reply:
It is done in a similar fashion as for kafka, although I agree that it looks a bit odd.

Reply:
Also your approach does not looks perfect, since `tables` has a special meaning then.

Reply:
Plus, currently only simple table names is supported (and maybe it will be even more limited in yaml format).

So if we want to change this, I'm suggesting to fix kafka first.
Although thinking about this a little bit more, and I guess kafka-like syntax may be expected by some users (who uses kafka), so maybe we can address this later.

As for the config, how about?

```xml
<rocksdb>
  <settings>
    <max_background_jobs>1</max_background_jobs>
  </settings>
  <tables>
    <table>
      <name>data</name>
      <settings>
        <max_background_jobs>1</max_background_jobs>
      </settings>
    </table>
  </tables>
```

*Although, looks a little bit verbose to me, but hay, this is XML anyway*

Reply:
Yes, this will be perfect. I don't care about verbosity there at all :)

Also it does not matter if config for Kafka has beed made in inferior way - we can rework it and add support for better way.

Reply:
> this is XML anyway

Since version 21.8 you can also write config in YAML :)